webnn/resources/test_data/relu.json

```diff
@@ -0,0 +1,324 @@
+{
+  "tests": [ // relu the same input test data of 1D ~ 5D shapes
+    {
+      "name": "relu 1D input",
+      "inputs": {
```

Comment:
Here `inputs` is also dictionary, which has "x" input to align with `relu(MLOperand x);`, the key 'x' is as input operand name by code 
https://github.com/BruceDai/wpt/blob/3edeb4c341e83794125804a06b8ba890a71bcc6b/webnn/relu.https.any.js#L13